package.json

```diff
@@ -54,14 +54,14 @@
     "@atlaskit/smart-card": "^12.6.2",
     "@material-ui/core": "^3.9.3",
     "cozy-bar": "^7.7.8",
-    "cozy-client": "^9.10.0",
+    "cozy-client": "9.10.0",
     "cozy-device-helper": "^1.8.0",
     "cozy-doctypes": "^1.70.0",
-    "cozy-flags": "^2.1.0",
+    "cozy-flags": "2.1.0",
     "cozy-realtime": "^3.2.3",
     "cozy-scripts": "^2.0.2",
     "cozy-sharing": "^1.5.2",
-    "cozy-ui": "^30.5.0",
+    "cozy-ui": "30.5.0",
```

Comment:
Weird to fix a few libs but not all, no?

Reply:
Dès que cette PR est actée, j'applique https://github.com/cozy/cozy-notes/pull/85 qui va étendre à tout le reste